contracts/Gauge.sol

```diff
@@ -52,15 +52,12 @@ contract Gauge is BaseGauge, IGauge {
     @dev rewards are queued when an account `_updateReward`.
     */
     uint256 public queuedVeYfiRewards;
-    uint256 private _totalSupply;
-    mapping(address => Balance) private _balances;
+    mapping(address => uint256) private _boostedBalances;
     mapping(address => mapping(address => Appoved)) public approvedTo;
 
     //// @notice list of extraRewards pool.
     address[] public extraRewards;
 
-    event Staked(address indexed user, uint256 amount);
-    event Withdrawn(address indexed user, uint256 amount);
     event AddedExtraReward(address indexed reward);
     event DeletedExtraRewards(address[] rewards);
     event RemovedExtraReward(address indexed reward);
```

Comment:
Maybe I am missing something but why do we have two events for similar actions?

Please, cmiiw, Is it possible to use the `DeletedExtraRewards` event and pass an array with only one item where we are using the `RemovedExtraReward` event?

Reply:
That can work

Reply:
That will need some memory manipulation

Reply:
> Maybe I am missing something but why do we have two events for similar actions?
> 
> Please, cmiiw, Is it possible to use the `DeletedExtraRewards` event and pass an array with only one item where we are using the `RemovedExtraReward` event?

I think that it is more readable, and we can remove the `RemovedExtraReward` event.